jOOQ-examples/jOOQ-flyway-ddl-example/pom.xml

```diff
@@ -0,0 +1,111 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.jooq</groupId>
+    <artifactId>jooq-flyway-ddl-example</artifactId>
+    <version>1.0</version>
+    <name>jOOQ Flyway/DDL Example</name>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.jooq.org/inc/LICENSE.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <org.springframework.version>3.2.6.RELEASE</org.springframework.version>
+        <org.jooq.version>3.12.0-SNAPSHOT</org.jooq.version>
+    </properties>
+
+    <dependencies>
+
+        <!-- Database access -->
+        <dependency>
+            <groupId>org.jooq</groupId>
+            <artifactId>jooq</artifactId>
+            <version>${org.jooq.version}</version>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
```

Comment:
Great idea to create an example project for this. It would be cool to actually use the flyway plugin, too and run a quick query on the resulting database. I'll make something up. I'd like to play around with Flyway's programmatic API for this, to create an in-memory H2 database.